store/v2/commitment/store.go

```diff
@@ -297,6 +283,7 @@ func (c *CommitStore) Get(storeKey []byte, version uint64, key []byte) ([]byte,
 	return bz, nil
 }
 
+// Prune implements store.PausablePruner.
```

Comment:
Actually this implements `store.Pruner`

```suggestion
// Prune implements store.Pruner.
```